docs/sphinx/source/whatsnew/v0.11.0.rst

```diff
@@ -9,6 +9,9 @@ Breaking changes
 ~~~~~~~~~~~~~~~~
 * The deprecated ``pvlib.modelchain.basic_chain`` has now been removed. (:pull:`1862`)
 * Remove the `poa_horizontal_ratio` function and all of its references. (:issue:`1697`, :pull:`2021`)
+* ``pvlib.iotools.read_srml_month_from_solardat()`` was deprecated in v0.10.0 and has
```

Comment:
```suggestion
* ``pvlib.iotools.read_srml_month_from_solardat`` was deprecated in v0.10.0 and has
```

I feel like being overly nitpicky here, but just for consistency with the other new code-like references of the same function.